src/pages/academy/blog/_all-articles.tsx

```diff
@@ -14,21 +16,47 @@ type AllArticlesProps = {
 export type ItemType = ArticleDataType[0]
 
 const AllArticles = ({ article_data }: AllArticlesProps) => {
+    const [is_mobile] = useBrowserResize()
+    const [is_tablet] = useBrowserResize(1333)
+    const [current_page, setCurrentPage] = useState(1)
+    const desktop_max_posts = 19
+    const tablet_max_posts = 9
```

Comment:
Same here

Reply:
For the tablet we will discuss with content team and QA to see which number is the best.